pages/_app.js

```diff
@@ -1,16 +1,91 @@
 import App from 'next/app';
 import Head from 'next/head';
-import React from 'react';
+import React, { useEffect } from 'react';
 import { GlobalStyle } from '@catho/quantum';
+import Image from 'next/image'
```

Comment:
Doubt: why is this necessary? a tag img from html is not sufficient?

Reply:
The image has different behaviors when called from the img tag and when using this component.
With this component, the size of the image changes according to the size of the screen. I had trouble showing the logo and found this solution using the Image component. However, for the purpose of this project and after changing the folders, it now works with <img>